quarkus.ls.ext/com.redhat.quarkus.ls/src/main/resources/com/redhat/quarkus/snippets/quarkus-java.json

```diff
@@ -46,5 +46,23 @@
 		"context": {
 			"type": "io.quarkus.test.junit.SubstrateTest"
 		}
+	},
+	"Quarkus - new integration test class": {
+		"prefix": "qitrc",
+		"body": [
+			"package ${1:packagename};",
+			"",
+			"import io.quarkus.test.junit.QuarkusIntegrationTest;",
+			"",
+			"@QuarkusIntegrationTest",
+			"public class ${TM_FILENAME_BASE} extends ${2:${TM_FILENAME_BASE/^(.*)IT/$1/}Test} {",
+			"",
+			"\t// Execute the same tests, but in native mode.",
+			"}"
+		],
+		"description": "Quarkus native test resource class",
```

Comment:
I have never tried, but I wonder if we could add in the description teh following markdown content

See [here](https://quarkus.io/guides/getting-started-testing#quarkus-integration-test) for more informations.

Reply:
I tried it, and it didn't work :-(. We should make an issue on lsp4mp to add that, since I think it could be helpful.

Reply:
See https://github.com/eclipse/lsp4mp/issues/305